build.gradle

```diff
@@ -28,6 +28,14 @@ ext.ext_classpath = ['com.google.code.gson:gson:2.2.4',
   'io.dropwizard.metrics:metrics-core:3.1.2']
 ext.target_dir = "$distsDir"
 ext.target_java8_dir = "${ext.target_dir}/java8"
+ext.target_javadoc_dir = "${ext.target_dir}/docs/javadoc"
+
+if (JavaVersion.current().isJava8Compatible()) {
+  tasks.withType(Javadoc) {
+    options.addStringOption('Xdoclint:none', '-quiet')
```

Comment:
Is Xdoclint:none still required?  I did a lot of work cleaning up things but maybe I didn't get everything.  Were there many complaints?  If possible/sensible, it would seem better to just fix them.  If that's a good idea but we really don't want to tackle that at this instant, please create another jira to track doing that after we finally convert to gradle.